Merging with updated zones

HappyVlane opened the pull request:
[Pull Request Web Preview](https://INSERT_USERNAME.github.io/dark-souls-3-cheat-sheet/)
# Changes

Added steps to zones past Cathedral and some small changes in other zones.

HappyVlane committed:
Added steps to all zones past Cathedral

Some parts of the old steps were rewritten to create better flow and new steps were added to create a guided experience.

HappyVlane committed:
Updated Undead Settlement

HappyVlane committed:
Updated several categories.

ZKjellberg commented:
Working to add this data in, but I'm a bit occupied as I'm currently in the process of moving from Germany back to the U.S.

First: [Here is the link to preview this Pull Request](https://happyvlane.github.io/dark-souls-3-cheat-sheet/). In future PR, please change the link of your post to have your username.

I'm having issues with merging as GitHub only shows as '529 additions, 240 deletions' without identifying the changes.

I see the following number increases in zone counters:
1. (1) Undead Settlement
2. (59) Farron Keep
3. (21) Catacombs
4. (25) Smouldering Lake
5. (50) Irithyll
6. (6) Annor Londo
7. (25) Irithyll Dungeon
8. (17) Profanned Capital
9. (6) Consumed Garden
10. (8) Untended Graves
11. (24) Archdragon Peak
12. (24) Lothric Castle
13. (25) Grand Archives

For a total of about (291) new bullets. Wow. I thought this was just Farron Keep but you powered through and added a massive amount of content. This will take some time to merge, but sounds wonderful to help add the missing zones for Issue #114, the 100% guide. 

@ElectroSpecter A heads up on all the work being done.

ZKjellberg commented:
First change suggestion: [Ran the website through W3C Validator and it found some HTML errors.](https://validator.w3.org/nu/?doc=https%3A%2F%2Fhappyvlane.github.io%2Fdark-souls-3-cheat-sheet%2F)

Ignore Error 1. This is Apple using non-standard code for their FavIcon.

Error 2-5 is an extra " causing confusion.

For 6-11, swap the URL with: http://darksouls3.wiki.fextralife.com/Titanite%20Shard
HTML does not like using whitespace.

For 12-13, it appears you have two extra <\li> tags.

HappyVlane committed:
Fixed validation errors

HappyVlane commented:
I've fixed the validation errors and ran it again, now it only shows Error 1.

You don't have to hurry with the merging. I'm currently in the process of running through the game again and making sure that the directions are understandable and adding items I might have missed (@ElectroSpecter already sent me some edits for Farron Keep).

ZKjellberg commented:
Sounds good. Nice to see quick fixes and happy to hear you are collaborating with Electro.

How are you making your edits? Your last edit shows as _357 additions, 357 deletions_ and GitHub web is unable to show the changes. I find it a little troubling to merge the changes without being able to see the core changes. I expect this has to do with your editor reformatting lines. I'm just wondering how to best merge your changes. I'll probably need to do a merge with an external tool and review all the changes line by line.

HappyVlane commented:
I haven't done the editing directly on github if that's what you mean. I did them locally and copy pasted the entire thing into the index.html that is on github (bad practice I assume).

In the future I will do the edits directly on github. Sorry for the inconvenience. Like I said, not too familiar with the site yet.

ZKjellberg commented:
Ah, I see. I expect when you copy your code over, it causes some formatting changes.

The GitHub web editor can be challenging, especially with a file as large as this. I find it frustrating to make simple tweaks.

Git can be awkward to learn, but I would recommend using a client like [GitKraken](https://www.gitkraken.com) to checkout the code and push up your changes. This will allow you to modify the files locally before pushing up your changes.

If this is too troublesome, you can continue your current workflow and we can address the issue of merging later on. Otherwise, I'm happy to help explain the process further.

HappyVlane committed:
Quick addition to Farron Keep

HappyVlane commented:
I've installed GitKraken and created a short commit with it to test it. I'll be using it from now on.

HappyVlane committed:
Update to Farron Keep

Farron Keep was updated with the edits @ElectroSpecter provided and some minor edits in Undead Settlement and Road of Sacrifices.

HappyVlane committed:
Update to several sections in index.html

Mostly clarifications for directions and fixed some errors for item amounts.

HappyVlane commented:
Just wanted to inform you that I just finished my playthrough using the guide and edited the index.html to reflect some, mostly minor, changes.

Until the DLC releases I'm done unless there are some glaring errors.

ZKjellberg commented:
Wonderful work @HappyVlane. 👍 

I'm nearly settled after my move, so I'll be looking to start the merge this week. Thanks!

HappyVlane committed:
Small change in Grand Archives

Changed "Greatax wielding foe" to Lion Knight Albert.

ZKjellberg commented:
@HappyVlane I still have this on my list to merge, but I'm currently still swamped with my move so I'll try to get this started when I have time. Due to the way you made your changes, I'll need to view the changes line by line to see what is just different formatting from the copy & paste or new content.

Just to confirm, you only added content to the zones and haven't touched things like Achievements or Javascript code? It'll be easier just to go zone by zone to review your changes so I can start the process and piecemeal it as I have time.

HappyVlane commented:
I have only changed/added the zone entries in the index.html. I've not touched any of the other code in the file or any other files.

HappyVlane committed:
Small change to Orbeck's questline

Changed "Slumbering Dragon Ring" to "Slumbering Dragoncrest Ring" and added "Spook" to the required spells for the ring

ZKjellberg commented:
Merged the code today. Shockingly the Merge wasn't bad at all. GitKraken made it really easy.

@HappyVlane Thanks for your hard work. this is really fantastic and wonderful to have integrated. I really appreciate the help. 👍 

The only change I made after merging your code was to fix some of the indentations so they were uniform with the rest of the code. I think Issue #114 might be finished now.
